x-pack/test/scalability/runner.ts

```diff
@@ -28,6 +30,7 @@ export async function ScalabilityTestRunner(
       args: [
         'gatling:test',
         '-q',
+        `-Dgatling.core.outputDirectoryBaseName=${fileName}`,
```

Comment:
Making report directory name to contain journey name
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/10977896/205739558-dc20b64a-d828-4888-8259-305aadee0cd6.png">